tests/env_file_test.conf

```diff
@@ -1,3 +1,4 @@
+# Ignored comment
 VAR_OTHER=file
-
+# Comment
```

Comment:
I'm not really sure why we support those, its just what was in the original feature. I have no opinion.